frontend/src/App.tsx

```diff
@@ -14,13 +14,17 @@ function App() {
 
   const [nodeManager, setNodeManager] = useState<NodeManager>();
 
-  const [newPubkey, setNewPubkey] = useState("...")
+  const [newPubkey, setNewPubkey] = useState("")
 
   // Send state
   const [txid, setTxid] = useState("...")
   const [amount, setAmount] = useState("")
   const [destinationAddress, setDestinationAddress] = useState("")
 
+// TODO make these configurable via input box
+  const [proxyAddress, setProxyAddress] = useState("ws://127.0.0.1:3001")
+  const [connectPeer, setConnectPeer] = useState("0307a40e17f451e1030401ef8dd698dac0a4a37bae2e904a337aeeefd39a78b2b9@127.0.0.1:4000")
```

Comment:
should we do this TODO?

Reply:
Added an input box for peer connection string